lib/celluloid/tasks/threaded.rb

```diff
@@ -0,0 +1,55 @@
+module Celluloid
+  class Task
+    # Tasks with a Thread backend
+    class Threaded < Task
```

Comment:
For consistency and following precedent. There is `Task::` and now `Group::` which are both ( like `Server::` were, moving from one to two to many ) in their own directories.

I hesitated to change the directory names to singular. I think plural may be better, but wasn't sure of changing `server/` to match precedent at every point.

Both tasks and groups are organized by type within one namespace rather than diluting the general namespace. And we have `PooledFiber` dropping into `Task` as well, but in its own gem. Something about a gem adding to the general namespace to add a specific type of something we have several of felt like we officially and markedly went over the point where there is one type of task, one internal pool and no other way around it. Now we encourage overriding those very loudly.